be/tool/beck.c

```diff
@@ -419,18 +425,20 @@ static struct btype bt[] = {
 #undef _B
 
 enum {
-	MAX_GEN    	 =      256,
-	MAX_SCAN_QUEUED	 = 10000000,
-	MAX_QUEUED  	 =  1000000,
-	MAX_REC_SIZE     =  64*1024,
+	MAX_GEN    	         = 256,
+	MAX_SCAN_QUEUED	         = 10000000,
+	MAX_QUEUED  	         = 1000000,
+	MAX_REC_SIZE             = 64*1024,
 	/**
-	 * This value is arrived on the basis of max time difference between
-	 * mkfs run on local and remote node and the assumption that time
-	 * difference between nodes is negligible.
+	 * The value in MAX_GEN_DIFF_SEC is arrived on the basis of max time
+	 * difference between mkfs run on local and remote node and the
+	 * assumption that clock time delta between nodes is negligible.
 	 */
-	MAX_GEN_DIFF_SEC =       30,
-	MAX_KEY_LEN      =      256,
-	MAX_VALUE_LEN    =      256
+	MAX_GEN_DIFF_SEC         = 30,
+	MAX_KEY_LEN              = 256,
+	MAX_VALUE_LEN            = 256,
+	DEFAULT_BE_SEG_LOAD_ADDR = 0x400000100000,
+	DEFAULT_BE_MAX_TX_REG_SZ = 46137344
```

Comment:
To be more readable, this number can be written as:
DEFAULT_BE_MAX_TX_REG_SZ = 44 * 1024 * 1024

Also, can you please explain where this constant comes from. 
Why is 44MB?